content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+
+**Note**: There is no `read-Only` permission as it is defined by the lack of both the `admin` and `standard` permissions for a role.
+
+## Advanced Permissions
+
+By default, existing users are already associated with one of the three out-of-the-box Datadog Admin, Standard, or Read-Only Roles, so all users already have permissions to read all data types, and Admin or Standard users already have write permissions on assets.
+
+**Note**: When adding a new custom role to a user, make sure to remove the out-of-the-box Datadog role associated with that user order to enforce the new role permissions.
+
+In addition of the general permissions, it is possible to define more granular permissions for specific assets or data types.
+Permissions can be either global or scoped to a subset of elements.
+Find below the details of these options and the impact they have on each available permission.
+
+### Dashboards
+
+| Name                    | Description                             | Scopable |
+| ----------------------- | --------------------------------------- | -------- |
+| dashboards_read         | Ability to view dashboards              | false    |
+| dashboards_write        | Ability to create and change dashboards | false    |
+| dashboards_public_share | Ability to share dashboards externally  | false    |
+
+### Monitors
+
+| Name              | Description                                  | Scopable |
+| ----------------- | -------------------------------------------- | -------- |
+| monitors_read     | Ability to view monitors                     | false    |
+| monitors_write    | Ability to change, mute, and delete monitors | false    |
+| monitors_downtime | Ability to set downtimes for your monitors   | false    |
+
+### Log Management
+
+| Name                         | Description                                | Scopable |
+| ---------------------------- | ------------------------------------------ | -------- |
+| logs_read_index_data         | Read a subset of all log indexes           | true     |
+| logs_modify_indexes          | Update the definition of log indexes       | false    |
+| logs_live_tail               | Access the live tail feature               | false    |
+| logs_write_exclusion_filters | Update a subset of the exclusion filters   | true     |
+| logs_write_pipelines         | Update a subset of the log pipelines       | true     |
+| logs_write_processors        | Update the log processors in an index      | true     |
+| logs_write_archives          | Update the external archives configuration | false    |
+| logs_public_config_api       | Access the Logs Public Config API (r/w)    | false    |
+| logs_generate_metrics        | Access the Generate Metrics feature        | false    |
+
```

Comment:
i guess we'll add the `logs_read_data` permission later?

Reply:
The plan is to indeed add it in the next PR when we add information about Query based RBAC